converter/fluka/helper_parsers/beam_parser.py

```diff
@@ -1,6 +1,8 @@
 from math import cos, atan, pi
 from dataclasses import dataclass
 from enum import Enum
+from converter.common import convert_beam_energy
+from converter.shieldhit.parser import PARTICLE_DICT
```

Comment:
Can you move PARTICLE_DICT to some levels up ? importing in Fluka something that belongs to SHIELD-HIT12A is bad architectural choice